### PR TITLE
adapt to override syntax changes and new tune file location

### PR DESCRIPTION
**Pull request by kraj**

### CODE REVIEW

File: recipes-devtools/sam-ba/sam-ba_3.3.1.bb [hunk 1]
```diff
@@ -4,7 +4,7 @@ LICENSE = "GPL-2.0"
 
 LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
-SRC_URI = "https://github.com/atmelcorp/${BPN}/releases/download/v${PV}/${BPN}_${PV}-linux_x86_64.tar.gz"
+SRC_URI = "https://github.com/atmelcorp/${BPN}/releases/download/v${PV}/${BPN}_${PV}-linux:x86_64.tar.gz"
```
Comment (codrin989) [inline recipes-devtools/sam-ba/sam-ba_3.3.1.bb:7]: hi @kraj  . I don't think this is correct, did you use a script to replace '_' with ':'?

Comment (kraj) [reply recipes-devtools/sam-ba/sam-ba_3.3.1.bb]: ah that was an oversight fixed it now and yes there is a conversion script in oe-core scripts/contrib/convert-overrides.py which helps but the results need to be reviewed.

**Comment by kraj** [comment]
ping on this, as  meta-atmel is broken on master for many days now.

**Comment by texierp** [comment]
Hi @kraj 

After a quick check, it seems that commit in [1] is needed.
Can you grab this one for this PR ?

[1] - https://github.com/YoeDistro/meta-atmel/pull/2

**Review by texierp — APPROVED** [review]

File: licenses/ATMEL_LLA_SAM-BA [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+
+THIS AGREEMENT IS THE ENTIRE AND EXCLUSIVE AGREEMENT BETWEEN ATMEL AND DEVELOPER AND SUPERSEDES ALL PRIOR ORAL AND WRITTEN AGREEMENTS AND COMMUNICATIONS BETWEEN THE PARTIES PERTAINING TO THE SUBJECT MATTER OF THIS AGREEMENT.  NO DIFFERENT OR ADDITIONAL TERMS WILL BE ENFORCEABLE AGAINST ATMEL UNLESS ATMEL GIVES ITS EXPRESS WRITTEN CONSENT, INCLUDING AN EXPRESS WAIVER OF THE TERMS OF THIS AGREEMENT.
+
+
+Atmel Corporation
+1600 Technology Drive
+San Jose, CA 95110
+http://www.atmel.com
+
+
```
Comment (codrin989) [inline licenses/ATMEL_LLA_SAM-BA:32]: could you please tell me what exactly changed here?

**Comment by codrin989** [comment]
the changes seem fine. I tried to build an image however, and after some changes in conf/local.conf.sample, it fails at perl-native. The build configuration I used is:
BB_VERSION           = "1.51.1"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "universal"
TARGET_SYS           = "arm-poky-linux-gnueabi"
MACHINE              = "sama5d27-wlsom1-ek-sd"
DISTRO               = "poky-atmel"
DISTRO_VERSION       = "3.3+snapshot-a5b257006b2c480d86e09909cdafb3c8ba05b863"
TUNE_FEATURES        = "arm vfp cortexa5 neon vfpv4 thumb callconvention-hard"
TARGET_FPU           = "hard"
meta                 
meta-poky            
meta-yocto-bsp       = "master:a5b257006b2c480d86e09909cdafb3c8ba05b863"
meta-oe              
meta-networking      
meta-webserver       
meta-python          
meta-initramfs       = "master:7e435caa8e3f1cb1f1034cad76b75575328ba511"
meta-atmel           = "YoeDistro-yoe/mut:e088eb50752fc60b72ead480adefd96acf4ab092"
meta-multimedia      = "master:7e435caa8e3f1cb1f1034cad76b75575328ba511"

File: licenses/ATMEL_LLA_SAM-BA (hunk 2 above)
Comment (kraj) [reply licenses/ATMEL_LLA_SAM-BA]: that was unwanted whitespace changes introduced by editor, I have fixed it and uploaded new pull. Thanks for reporting it.

**Comment by kraj** [comment]
ping ^2 on this pull, it has been pending for more than 3 months now and master branch is broken without it. Let me know if there are any issues with it.

**Comment by kraj** [comment]
Can someone look into this pull please ??

**Comment by noglitch** [comment]
@kraj, @texierp ,
Thanks a lot for your help on this PR. Indeed this PR waited too long, sorry about that.
Best regards,
  Nicolas

**Pull request merged**